readme.md

```diff
@@ -2,7 +2,7 @@
 
 > [Memoize](https://en.wikipedia.org/wiki/Memoization) functions - An optimization used to speed up consecutive function calls by caching the result of calls with identical input
 
-Memory is automatically released when an item expires.
+The input is cached by reference when it’s not an object and when the argument is only one; it’s cached by value in all other cases (where `JSON.stringify` supports it). In the first case, memory is automatically released when an item expires (thanks to `WeakMap`).
```

Comment:
> The input is cached by reference when it’s not an object

This can never be true, as "not object" means a primitive, which cannot be a reference.

Reply:
lol true

Reply:
I see what you meant earlier now